streams/src/main/java/org/apache/kafka/streams/processor/internals/metrics/StreamsMetricsImpl.java

```diff
@@ -294,14 +293,6 @@ public final void removeAllThreadLevelSensors(final String threadId) {
         return tagMap;
     }
 
-    public Map<String, String> bufferLevelTagMap(final String threadId,
```

Comment:
This public function on an internal class is never used in our code base, so I assume it is safe to delete.

Reply:
\cc @cadonna Any though on this? (Similar below)

Reply:
That is fine! Apparently I missed this method when I removed the old Streams metrics structure. Thanks @jlprat !